test/parse/locator.vows.js

```diff
@@ -71,6 +73,6 @@ var xml= [
     var doc = parser.parseFromString('<root>\n\t<err</root>', 'text/html');
     var root = doc.documentElement;
     var textNode = root.firstChild;
-	assert.isTrue(/\n@#\[line\:2,col\:2\]/.test(error.join(' ')),'line,col must record:'+JSON.stringify(error));
+	assert(/\n@#\[line\:2,col\:2\]/.test(error.join(' ')),'line,col must record:'+JSON.stringify(error));
```

Comment:
A bit hard to read & understand, how about this idea:

```suggestion
	assert(/\n@#\[line\:2,col\:2\]/.test(error.join(' ')), 'line,col must record: ' + JSON.stringify(error));
```

or maybe even this:

```suggestion
	assert(/\n@#\[line\:2,col\:2\]/.test(error.join(' ')), `line,col must record: ${JSON.stringify(error)}`);
```

or one step further, like what I think Prettier would do:

```suggestion
        assert(
          /\n@#\[line\:2,col\:2\]/.test(error.join(' ')),
          `line,col must record: ${JSON.stringify(error)}`
        );
```